### PR TITLE
Fix: Changes in bigquey for project-id

### PR DESCRIPTION
**Pull request by NiharDoshi99**
### Describe your changes :

I worked in big-query for adding multiple project-ids
Fix: #8462 

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

### CODE REVIEW

**Review by pmbrull — COMMENTED** [review]
This will require some migration changes right? For existing BigQuery connections we should update the JSON serviceConnection to treat the Project ID as a list

**Comment by NiharDoshi99** [comment]
> This will require some migration changes right? For existing BigQuery connections we should update the JSON serviceConnection to treat the Project ID as a list

File: ingestion/src/metadata/utils/source_connections.py [hunk 1]
```diff
@@ -403,21 +403,18 @@ def _(connection: HiveConnection):
...
-            not connection.credentials.gcsConfig.privateKey
-            and connection.credentials.gcsConfig.projectId
-        ):
-            # Setting environment variable based on project id given by user / set in ADC
-            project_id = connection.credentials.gcsConfig.projectId
-            os.environ["GOOGLE_CLOUD_PROJECT"] = project_id
-        return f"{connection.scheme.value}://{project_id}"
-    return f"{connection.scheme.value}://"
+        for project_id in connection.credentials.gcsConfig.projectId:
+            if not project_id:
```
Comment (ayush-shah) [inline ingestion/src/metadata/utils/source_connections.py:409]: we can remove this condition i guess @NiharDoshi99

File: ingestion/src/metadata/utils/credentials.py [hunk 2]
```diff
@@ -76,13 +76,13 @@ def build_google_credentials_dict(gcs_values: GCSValues) -> Dict[str, str]:
     :param gcs_values: GCS credentials
     :return: Dictionary with credentials
     """
-
+    print("gcs_values", gcs_values)
```
Comment (pmbrull) [inline ingestion/src/metadata/utils/credentials.py:79]:
```suggestion
```

Kill all the prints 🔪

Comment (NiharDoshi99) [reply ingestion/src/metadata/utils/credentials.py:79]: yes, my bad 😞

**Comment by ayush-shah** [comment]
should we add docs around this as well once along with the cahnges?

**Review by pmbrull — APPROVED** [review]
thanks!

**Comment by ayush-shah** [comment]
@NiharDoshi99 please create a separate issue to add docs around ingestion of multiple project ids

**Comment by NiharDoshi99** [comment]
> @NiharDoshi99 please create a separate issue to add docs around ingestion of multiple project ids

Yes sure will add @ayush-shah

**Review by ayush-shah — APPROVED** [review]

**Pull request merged**